pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -389,7 +389,6 @@ func Newk8sOrchestrator(ctx context.Context, controllerClusterFlavor cnstypes.Cn
 
 func getReleasedVanillaFSS() map[string]struct{} {
 	return map[string]struct{}{
-		common.CSIMigration:                   {},
```

Comment:
Please remove FSS from pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator_test.go, those are missed in this diff